docs/pages/api-docs/data-grid/grid-col-def.md

```diff
@@ -12,30 +12,31 @@ import { GridColDef } from '@material-ui/data-grid';
+| <span class="prop-name optional">renderHeader<sup><abbr title="optional">?</abbr></sup></span>      | <span class="prop-type">(params: GridColumnHeaderParams) =&gt; ReactNode</span>                    |                                                  | Allows to render a component in the column header cell.                                                         |
+| <span class="prop-name optional">resizable<sup><abbr title="optional">?</abbr></sup></span>         | <span class="prop-type">boolean</span>                                                             | <span class="prop-default">true<br /></span>     | If `true`, the column is resizable.                                                                             |
+| <span class="prop-name optional">sortComparator<sup><abbr title="optional">?</abbr></sup></span>    | <span class="prop-type">GridComparatorFn</span>                                                    |                                                  | A comparator function used to sort rows.                                                                        |
+| <span class="prop-name optional">sortable<sup><abbr title="optional">?</abbr></sup></span>          | <span class="prop-type">boolean</span>                                                             | <span class="prop-default">true<br /></span>     | If `true`, the column is sortable.                                                                              |
+| <span class="prop-name optional">type<sup><abbr title="optional">?</abbr></sup></span>              | <span class="prop-type">string</span>                                                              | <span class="prop-default">'string'<br /></span> | Type allows to merge this object with a default definition [GridColDef](/api/data-grid/grid-col-def/).          |
+| <span class="prop-name optional">valueFormatter<sup><abbr title="optional">?</abbr></sup></span>    | <span class="prop-type">(params: GridValueFormatterParams) =&gt; GridCellValue</span>              |                                                  | Function that allows to apply a formatter before rendering its value.                                           |
+| <span class="prop-name optional">valueGetter<sup><abbr title="optional">?</abbr></sup></span>       | <span class="prop-type">(params: GridValueGetterParams) =&gt; GridCellValue</span>                 |                                                  | Function that allows to get a specific data instead of field to render in the cell.                             |
+| <span class="prop-name optional">valueOptions<sup><abbr title="optional">?</abbr></sup></span>      | <span class="prop-type">string \| []</span>                                                        |                                                  | To be used in combination with `type: 'singleSelect'`. This is an array of the possible cell values and labels. |
```

Comment:
This is outdated.

Reply:
It is the same as in the `gridColDef`. Maybe we can say it is `This is an array of the possible cell string values and/or labels.` ?

Reply:
The description is already up to date. I was making a reference to the type, that is outdated, compared to the implementation:

<img width="1037" alt="Capture d’écran 2021-06-29 à 15 54 53" src="https://user-images.githubusercontent.com/3165635/123810081-6bf78180-d8f2-11eb-8077-22447797bb79.png">

Reply:
Hmm seems to be a bug with the docs generation script. `Array<string | { value: any; label: string }>` is converted to `string \| []`

Reply:
This will be fixed by #2003.